resources/adobe-experience-platform-mini-programs-sdk/README.md

```diff
@@ -24,14 +24,14 @@ The SDK automatically collects out-of-the-box lifecycle metrics and sends it to
 
 ### Identity
 
-The SDK generates an Adobe Analytics Visitor ID \(aid\) to identify program users.
+The SDK generates an Adobe Analytics Visitor ID (aid) to identify program users.
 
 {% hint style="warning" %}
-At this time, Experience Cloud ID \(ECID or MCID\) **are not** supported.
+At this time, Experience Cloud ID (ECID or MCID) **are not** supported.
 {% endhint %}
 
 {% hint style="info" %}
-To learn more about Analytics Visitor ID, see the [Adobe Analytics implementation guide](https://docs.adobe.com/content/help/en/analytics/implementation/javascript-implementation/unique-visitors/visid-analytics.html#concept_74F6B4B9B2FA415AB5D029A1F8F099BC).
+To learn more about Analytics Visitor ID, see the [Adobe Analytics implementation guide](https://experienceleague.adobe.com/docs/analytics/implementation/js/xdevice-visid/xdevice-connecting.html?lang=en).
```

Comment:
The page on this link says this is not the recommended method. Maybe we can clarify if this is the correct link?